README.md

```diff
@@ -79,14 +83,14 @@ Your answer...
 You clone yet another Tunr repo. Put the following commands in the correct order necessary to make the app run. Delete the one command that will not be used.
 
 ```
-$ rake db:drop
-$ rake db:create
-$ rails new . -d postgresql
-$ bundle install
 $ git clone git@github.com:ga-wdi-exercises/moar-tunr.git
+$ cd moar-tunr # I added this (we must be in the project directory for the next commands to work)
+$ bundle install
+$ rails new . -d postgresql
```

Comment:
@joe-gz Yes, but some of the apps that we were instructed to clone (like scribble) required the creation of a new app after cloning a bare git repository.  That's what I figured the quiz was getting at since this tripped up people in scribble.  I'd suggest the question be more specific since there weren't any explicit rules stating all tunr repos will always already have full git tracking of all files requires to run rails.